evm/app/deployer.js

```diff
@@ -49,7 +49,7 @@ module.exports = function Deployer(wallet, utils) {
       const encodedArgs = encodeArgs(contractArgs, compiled.abi)
 
       const txHash = await wallet.send({
-        gas: 2600000,
+        gas: 4000000,
```

Comment:
Yes! This could be fixable if we upgrade to Solidity 5.0. I've read that the ABIEncoderV2 is pretty expensive at the moment, but maybe that has been fixed in recent compiler versions.